sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/DoNotInstantiateSharedClassesTest.cs

```diff
@@ -19,7 +19,10 @@
  */
 
 extern alias csharp;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
is Linq used?

Reply:
in general, maybe use some plugin to cleanup unused usings

Reply:
We have to activate **S1128** also for unit tests.

Reply:
fair point, we should look into how to do that (activate rules for unit tests - I believe it's a setting in the RSPEC)